src/axom/inlet/Table.hpp

```diff
@@ -639,18 +663,56 @@ class Table : public Verifiable
     T result;
     // This needs to work transparently for both references to the underlying
     // internal table and references using the same path as the data file
-    if(axom::utilities::string::endsWith(m_name, ARRAY_GROUP_NAME))
+    if(axom::utilities::string::endsWith(m_name, CONTAINER_GROUP_NAME))
```

Comment:
```cpp
if( axom::utilities::string::endsWith(m_name, CONTAINER_GROUP_NAME) ) {...}
```

Minor: It might be better (w.r.t. readability and maintenance), to make this a utility function.
E..g.
```cpp
if( inlet::is_container_group (m_name) ) { .... }
```